plugins/modules/remote_management/lenovoxcc/xcc_redfish_command.py

```diff
@@ -0,0 +1,634 @@
+      resource_uri: "/redfish/v1/Managers/1/Oem/Lenovo/FoD/Keys"
+    register: result
+  - ansible.builtin.debug:
+      msg: "{{ result.redfish_facts.data_list }}"
+
+  - name: Update ComputeSystem property AssetTag via PatchResource command
+    community.general.xcc_redfish_command:
+      category: Raw
+      command: PatchResource
+      baseuri: "{{ baseuri }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+      resource_uri: "/redfish/v1/Systems/1"
+      request_body:
+        AssetTag: "new_asset_tag"
+
+  - name: Perform BootToBIOSSetup action via PostResource command
+    community.general.xcc_redfish_command:
+      category: Raw
+      command: PostResource
+      baseuri: "{{ baseuri }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+      resource_uri: "/redfish/v1/Systems/1/Actions/Oem/LenovoComputerSystem.BootToBIOSSetup"
+      request_body: {}
+
+  - name: Perform SecureBoot.ResetKeys action via PostResource command
+    community.general.xcc_redfish_command:
+      category: Raw
+      command: PostResource
+      baseuri: "{{ baseuri }}"
+      username: "{{ username }}"
+      password: "{{ password }}"
+      resource_uri: "/redfish/v1/Systems/1/SecureBoot/Actions/SecureBoot.ResetKeys"
+      request_body:
+        ResetKeysType: DeleteAllKeys
+
+'''
+
+RETURN = '''
+msg:
+    description: A message related to the performed action(s).
+    returned: when failure or action/update success
+    type: str
+    sample: "Action was successful"
+redfish_facts:
+    description: Resource content.
+    returned: when command == GetResource or command == GetCollectionResource
+    type: dict
+    sample: '{
+        "redfish_facts": {
+            "data": {
+                "@odata.etag": "\"3179bf00d69f25a8b3c\"",
+                "@odata.id": "/redfish/v1/Managers/1/NetworkProtocol/Oem/Lenovo/DNS",
+                "@odata.type": "#LenovoDNS.v1_0_0.LenovoDNS",
+                "DDNS": [
+                    {
+                        "DDNSEnable": true,
+                        "DomainName": "",
+                        "DomainNameSource": "DHCP"
+                    }
+                ],
+                "DNSEnable": true,
+                "Description": "This resource is used to represent a DNS resource for a Redfish implementation.",
+                "IPv4Address1": "10.103.62.178",
+                "IPv4Address2": "0.0.0.0",
+                "IPv4Address3": "0.0.0.0",
+                "IPv6Address1": "::",
+                "IPv6Address2": "::",
+                "IPv6Address3": "::",
+                "Id": "LenovoDNS",
+                "PreferredAddresstype": "IPv4"
+            },
+            "ret": true
+        }
+    }'
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils._text import to_native
+from ansible_collections.community.general.plugins.module_utils.redfish_utils import RedfishUtils
+
+
+class xcc_RedfishUtils(RedfishUtils):
```

Comment:
```suggestion
class XCCRedfishUtils(RedfishUtils):
```
In Python, convention is to use camel case for class names.

Reply:
Good suggestion, I have updated the code.